npm-shrinkwrap.json

```diff
@@ -169,6 +184,231 @@
       "from": "babel-generator@>=6.20.0 <7.0.0",
       "resolved": "https://registry.npmjs.org/babel-generator/-/babel-generator-6.20.0.tgz"
     },
+    "babel-helper-call-delegate": {
```

Comment:
We probably should regenerate this file from scratch with the aforementioned versions.